app/scripts/services/context/evalContextV3.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+angular
+  .module('wcagReporter')
+  .value('evalContextV3', {
```

Comment:
using `angular.constant` means these cannot be updated from any other place where this is dependency injected, so we should do that.

Reply:
Nice, will do

Reply:
Also made a constant of other contexts V1 && V2